CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 # CHANGELOG
 
-## On Demo
-### Features
-- Add BP Passport sheet
```

Comment:
We haven't moved it to `On Demo` or `Next Release`, is this expected?

Reply:
NVM, forgot to read the message 😅

Reply:
Since this is in progress and this will not be visible to nurses yet. Hence in the next release.